src/scene/materials/standard-material.js

```diff
@@ -970,9 +2083,9 @@ const defineAggProp = (prop) => {
 
     const setterFunc = function (value) {
         const oldValue = this[internalName];
-        if (!oldValue.equals(value)) {
+        if (!oldValue?.equals(value)) {
```

Comment:
What in this PR requires this function to be updated? Just curious.

Reply:
That probably wont be needed anymore actually I will double check

Reply:
So I found out this was because when defining the properties in the class with no value that is actually defining them with `undefined` and triggers the set handler before the default internal properties are set with `reset`.

Reply:
Oh...maybe consider adding a comment about that in the code then.